### PR TITLE
Add logic for homepage and show action.

### PR DESCRIPTION
**Pull request by JellyBellyDev**
ok guys now only the layout is missing! @elfo404 where are you? :P

### CODE REVIEW

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -13,17 +15,32 @@
...
      * @param ContributorsRepository $contributorsRepository
+     * @param ReaderInterface        $redisReader
+     * @param Poser                  $poser
      *
      * @return Response
      */
-    public function index(ContributorsRepository $contributorsRepository): Response
-    {
+    public function index(
+        $repository,
```
Comment (garak) [inline src/Controller/HomeController.php]: Can't we type-hint here?

Comment (JellyBellyDev) [reply src/Controller/HomeController.php]: sorry for my missing! :S

Comment (JellyBellyDev) [reply src/Controller/HomeController.php]: fixed!

File: tests/Controller/HomeControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class HomeControllerTest extends WebTestCase
+{
+    /**
+     * @dataProvider provider
+     * @param $path
+     */
+    public function testHome($path): void
```
Comment (garak) [inline tests/Controller/HomeControllerTest.php]: I guess we can type-hint here also :wink:

Comment (JellyBellyDev) [reply tests/Controller/HomeControllerTest.php]: fixed! thanks!

**Pull request merged**